### PR TITLE
Feature #r20 KatyaNemerovetska

### PR DESCRIPTION
**Pull request by Katy-dev**
dev
##Click Up

https://app.clickup.com/t/az1a7r

## Summary of issue

Implement our partners component

## Summary of change

Task has done

## Testing approach
manual

### CODE REVIEW

File: src/pages/home/index.tsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
 import React from 'react';
+import {Slider} from "../../components/slider";
```
Comment (andrew-melnychuk) [inline src/pages/home/index.tsx]: all components import in 1 line

File: src/pages/home/index.tsx [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 import styles from './home.module.scss';
 import { Header, Banner, BannerLower } from '../../components';
```
Comment (andrew-melnychuk) [inline src/pages/home/index.tsx]: here

File: src/pages/home/index.tsx [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
 
+
 const Home = () => {
   return (
     <>
       <Header />
       <Banner />
       <div className={styles.filter}>Filter</div>
       <div className={styles.about}>About</div>
+      <div className={styles.cta}>Call to action</div>
```
Comment (andrew-melnychuk) [inline src/pages/home/index.tsx]: delete this line

File: src/pages/home/index.tsx [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
+      <Slider />
       <BannerLower />
       <div className={styles.partners}>Partners</div>
```
Comment (andrew-melnychuk) [inline src/pages/home/index.tsx:15]: and this, here should be <Slider />

File: src/pages/home/index.tsx (hunk 1 above)
Comment (Katy-dev) [reply src/pages/home/index.tsx]: fixed

File: src/pages/home/index.tsx (hunk 2 above)
Comment (Katy-dev) [reply src/pages/home/index.tsx]: fixed

File: src/pages/home/index.tsx (hunk 3 above)
Comment (Katy-dev) [reply src/pages/home/index.tsx]: fixed

File: src/pages/home/index.tsx (hunk 4 above)
Comment (Katy-dev) [reply src/pages/home/index.tsx:15]: fixed

**Pull request closed without merging**